CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Changed
+
+- Rename `notify_user_async` celery task to `notify_user_about_new_alert_group` by @joeyorlando ([#2900](https://github.com/grafana/oncall/pull/2900))
```

Comment:
_nit_: do we want to include this in the changelog? looks like it's not a user-facing change